_faq/mentor_obligations.md

```diff
@@ -1,6 +1,12 @@
 ---
 question: "What are my obligations as a mentor during the GSoC?"
 ---
+coala has two mentors for every project:
+
+1. Primary mentor - responsible for the administrative tasks of the project.
+2. Secondary mentor - has little to no responsibility, but provides backup and
+code review for aspects of the project which are relevant to their expertise.
```

Comment:
indent according to normal markdown numbered list style.

Reply:
Sorry, it's unclear. What do you want me to do here ? I have used ordered list according to https://guides.github.com/features/mastering-markdown/#syntax
What do indent ?